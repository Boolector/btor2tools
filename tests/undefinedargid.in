2 and 1 1
