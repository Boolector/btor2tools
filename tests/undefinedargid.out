*** catbtor: parse error in 'undefinedargid.in' line 1: undefined argument id
